qa/tasks/cephfs/mount.py

```diff
@@ -569,13 +569,17 @@ def get_keyring_path(self):
 
     def get_key_from_keyfile(self):
         # XXX: don't call run_shell(), since CephFS might be unmounted.
-        keyring = self.client_remote.run(
-            args=['sudo', 'cat', self.client_keyring_path], stdout=StringIO(),
-            omit_sudo=False).stdout.getvalue()
+        keyring = self.client_remote.read_file(path=self.client_keyring_path,
+                                               stdout=StringIO())
```

Comment:
Actually, do we need to add `sudo=True` for this? You dropped that without explanation.